src/types/ConnectionProfile.ts

```diff
@@ -120,3 +189,49 @@ export function createConnectionProfile(
     certificateAuthorities: certificateAuthorities(networkSettings.tls, rootPath, org),
   };
 }
+
+export function createHyperledgerExplorerConnectionProfile(
+  networkSettings: NetworkSettings,
+  org: OrgConfig,
+  orgs: OrgConfig[],
+  channels: ChannelConfig[],
+): HyperledgerExplorerConnectionProfile {
+  const rootPath = "/tmp/crypto";
+  const peers = createPeers(org.name, true, networkSettings.tls, rootPath, orgs);
+  return {
+    name: `fablo-test-network-${org.name.toLowerCase()}`,
+    description: `Connection profile for Hyperledger Explorer in Fablo network`,
+    version: "1.0.0",
+    client: {
+      organization: org.name,
+      tlsEnable: networkSettings.tls,
+      enableAuthentication: true,
+      adminCredential: {
+        id: "admin",
+        password: "adminpw",
+      },
+      connection: {
+        timeout: {
+          peer: {
+            endorser: "300",
+          },
+          orderer: "300",
+        },
+      },
+    },
+    organizations: {
+      [org.name]: {
+        mspid: org.mspName,
+        adminPrivateKey: {
+          path: `${rootPath}/peerOrganizations/${org.domain}/users/Admin@${org.domain}/msp/keystore/priv-key.pem`,
+        },
+        peers: Object.keys(peers),
+        signedCert: {
+          path: `${rootPath}/peerOrganizations/${org.domain}/users/Admin@${org.domain}/msp/signcerts/Admin@${org.domain}-cert.pem`,
```

Comment:
hm... been thinking - we're using here absolute path, don't we?  
Maybe it's possible ti switch to relative?

Reply:
Yes, it's an absolute path. This one is for the explorer, so I guess we could do that, but why should we change it? Do you mean just here, in `createHyperledgerExplorerConnectionProfile` method, or in other methods form this file as well?

Reply:
I meant only in this place :).  
and why relative is in imo better: someone might want to do `fablo generate` and only use our generated files as a starting point and then customize them and include in their git repository.  
It that scenario he will have absolute paths in commited files.

Reply:
But bare in mind that it's not super important to fix all places. We had some problems with relative path to chaincodes and thats why they are absolute. It's just an attempt to limit such cases if possible.

Reply:
This path is used by the explorer, inside docker container, so if I get it right, here instead of `/tmp/crypto` we would end up with something close to `../../../../../../../` 😉   imo that would require adjusting as well, so I wouldn't worry about that.